staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -327,10 +333,26 @@ func (o *AuditBatchOptions) AddFlags(pluginName string, fs *pflag.FlagSet) {
 			"moment if ThrottleQPS was not utilized before. Only used in batch mode.")
 }
 
+type safeBackend struct {
+	audit.Backend
+}
```

Comment:
Looks like type safeBackend should be put in package `pluginbuffered`

Reply:
I initially had a separate package for this, but @tallclair convinced me this makes more sense here, as it is quite trivial. Since nothing in safeBackend is pluginbuffered-specific, I'd rather leave it here.

Reply:
Sure. Thanks.